deps/v8/tools/js2c.py

```diff
@@ -32,7 +32,7 @@
 # library.
 
 import os, re, sys, string
-import optparse
+import argparse
```

Comment:
It has been fixed with https://crrev.com/44bc918458481d60b08d5566f0f31a79e39b85d7 but not backported to 4.4. I will include the patch tomorrow.
Thanks @ofrobots.